### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by natjeremy**
Updating per ADO task 189157.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: yammer/manage-yammer-users/office-365-sign-in.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ After a user is connected and using Office 365 sign-in for Yammer, the user can
   
 ## How do I enable Office 365 sign-in for Yammer?
 
-No action is needed to enable Office 365 sign-in for Yammer.
+Yammer will detect when an Azure AD account exists for a user and use Office 365 sign-in for Yammer. It's strongly recommended that admins [Enforce Office 365 identity for Yammer users](../configure-your-yammer-network/enforce-office-365-identity.md).
```
Comment (JohanFreelancer9) [inline yammer/manage-yammer-users/office-365-sign-in.md:50]:
```suggestion
Yammer will detect when an Azure AD account exists for a user and use Office 365 sign-in for Yammer. We strongly recommend that admins [Enforce Office 365 identity for Yammer users](../configure-your-yammer-network/enforce-office-365-identity.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review  - @natjeremy, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by natjeremy** [comment]
@pamgreen-msft Can you merge this update please? There is another request coming soon with a large amount of changes. Thanks!

**Pull request merged**